From 8a1a3947a10d717391828d7a54cada51194bae94 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 5 Sep 2014 19:49:09 -0400
Subject: [ethernet] bnx2x: fix sparse warning

Message-id: <1409946615-5862-5-git-send-email-mschmidt@redhat.com>
Patchwork-id: 91546
O-Subject: [RHEL7.1 PATCH 04/70] bnx2x: fix sparse warning
Bugzilla: 1089402
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1089402

Upstream commit:
commit 8d448b86f5d7c859f48188802dcabf78719c00b6
Author: stephen hemminger <stephen@networkplumber.org>
Date:   Tue Jan 14 10:14:11 2014 -0800

    bnx2x: fix sparse warning

    Fix new sparse warning about function declared static.

    Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c
index c5ce99b..9b6b3d7 100644
--- a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c
+++ b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c
@@ -8602,8 +8602,8 @@ static void bnx2x_set_limiting_mode(struct link_params *params,
  }
 }
 
-int bnx2x_sfp_module_detection(struct bnx2x_phy *phy,
-          struct link_params *params)
+static int bnx2x_sfp_module_detection(struct bnx2x_phy *phy,
+          struct link_params *params)
 {
  struct bnx2x *bp = params->bp;
  u16 edc_mode;
-- 
1.7.1